tests/deployment/app_test.go

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
Is this pure kubernetes testing?
can this be under `tests/kubernetes/deployment/`